README.md

```diff
@@ -30,7 +30,7 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
https://github.com/Regiss05/To-do-list/blob/ebe115a1d4890cd0e5928d48ecf0b9d7c3148cef/src/index.js#L95-L112

- [ ] Geat job so far :clap: , however according to the project requirement, you should create one new JavaScript files which will contain methods related to the status updates (`completed`: `true` / `false`) then imported it as a module.